website/docs/r/stream_analytics_job.html.markdown

```diff
@@ -0,0 +1,83 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_stream_analytics_job"
+sidebar_current: "docs-azurerm-resource-stream-analytics-job"
+description: |-
+  Manages a Stream Analytics Job.
+---
+
+# azurerm_stream_analytics_job
+
+Manages a Stream Analytics Job.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+resource "azurerm_stream_analytics_job" "example" {
+  name                                     = "example-job"
+  resource_group_name                      = "${azurerm_resource_group.example.name}"
+  location                                 = "${azurerm_resource_group.example.location}"
+  compatibility_level                      = "1.1"
+  data_locale                              = "en-GB"
+  events_late_arrival_max_delay_in_seconds = 60
+  events_out_of_order_max_delay_in_seconds = 50
+  events_out_of_order_policy               = "Adjust"
+  output_error_policy                      = "Drop"
+  streaming_units                          = 3
+
+  transformation_query = <<QUERY
+    SELECT *
+    INTO [YourOutputAlias]
+    FROM [YourInputAlias]
+QUERY
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the Stream Analytics Job. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the Stream Analytics Job should exist. Changing this forces a new resource to be created.
+
+* `location` - (Required) The Azure Region in which the Resource Group exists. Changing this forces a new resource to be created.
+
+* `compatibility_level` - (Required) Specifies the compatibility level for this job - which controls certain runtime behaviors of the streaming job. Possible values are `1.0` and `1.1`.
```

Comment:
Should this be:

```suggestion
* `compatibility_level` - (Required) Specifies the compatibility level for this job - which controls certain runtime behaviors of the streaming job. Possible values are `1.0` and `1.1`.
```

as validation ensures only `1.1` is accepted

Reply:
this allows 1.0 too: https://github.com/terraform-providers/terraform-provider-azurerm/pull/3227/files#diff-62a129b7065e0241918acf51006d6a0eR45